start.sh

```diff
@@ -0,0 +1 @@
+rasa run actions & rasa run --credentials ./credentials.yml  --enable-api --auth-token XYZ123 --model ./models --endpoints ./endpoints.yml --cors "*"
```

Comment:
Where will the log for "rasa run actions" go? Perhaps redirect to terminal too, or to some file in /tmp?

Reply:
Fixed the logging